model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -114,6 +114,15 @@ data class ResolvedLicenseInfo(
             }
         }
 
+    /**
+     * Filter all licenses by applying the [licenseChoices].
+     */
+    fun filter(licenseChoices: List<LicenseChoice>): ResolvedLicenseInfo {
```

Comment:
Not sure about the function name: Is this really a filter operation? I think, the licenses are reduced by applying concrete choices. So would there be better names, e.g. _applyChoices()_, _selectLicenses()_, ...?